roles/ovirt-vm-infra/README.md

```diff
@@ -44,14 +46,20 @@ The `profile` dictionary can contain following attributes:
 | sockets            | UNDEF                 | Number of virtual CPUs sockets of the Virtual Machine.  |
 | disks              | UNDEF                 | Dictionary specifying the additional virtual machine disks. See below for more detailed description. |
 | nics               | UNDEF                 | List of dictionaries specifying the NICs of the virtual machine. See below for more detailed description.   |
-| ssh_key            | UNDEF                 | SSH key to be deployed to the virtual machine.                 |
-| domain             | UNDEF                 | The domain of the virtual machine.                         |
-| root_password      | UNDEF                 | The root password of the virtual machine.                      |
-| cloud_init_nics    | UNDEF                 | List of dictionaries representing network interafaces to be setup by cloud init. See below for more detailed description. |
 | high_availability  | UNDEF                 | Whether or not the node should be set highly available. |
 | storage_domain     | UNDEF                 | Name of the storage domain where all virtual machine disks should be created. Considered only when template is provided.|
 | state              | present               | Should the Virtual Machine be stopped, present or running.|
 
+Following attributes of `profile` dictionary are depracated and will be remoed in version 1.1,
+please use `cloud_init` parameter instead. Those parameters has precendece before `cloud_init` parameter,
+so please remove them when using `cloud_init`.
```

Comment:
Those parameters has precedence before `cloud_init` parameter not to break backward compatibility,
+so please remove them when using `cloud_init` parameter.

Reply:
done